### ISSUE TITLE
How to use for graph classification

### ISSUE THREAD

**Issue by PeterDeSOM** [THUDM/GraphMAE#52]
So far, all that I've found the method for Graph Clustering is actually for node clustering or not a fully unsupervised learning method. It means that they eventually need their label to train at the downstream task.

For example, it is;
- pretraining some datasets with the unlabeled using the contrastive learning method or others. (No output channel or number of classes to be clustered is required)
- then finetuning the dataset with its labels using the supervised learning method.

I'm finding a fully unsupervised learning method for Graph-Level Clustering/Classification. Or, I'd like to know how to employ your model in a fully unsupervised learning method to classify lots of graph data that cannot be labeled by human efforts.

**Question. Could you show me a little direction to use the embedding from your model for the Graph-Level Classification/Clustering without its labels such as KMeans at the downstream task?**

**Comment by THINK2TRY** [comment]
Hi, @PeterDeSOM  Thanks for your attention to our work! Actually, GraphMAE is an unsupervised representation learning method, which pre-trains GNNs and learns node-level and graph-level embeddings without relying on any labeled data. The embeddings can be directly applied to any downstream task like clustering without finetuning. 

In our experimental setting, we simply train a linear classifier rather than finetune the GNN to evaluate the quality of the learned embeddings. Graph clustering such as KMeans can also be used for evaluation.
